src/analysis/seminorm.lean

```diff
@@ -584,6 +590,59 @@ lemma comp_smul_apply (p : seminorm 𝕜 F) (f : E →ₗ[𝕜] F) (c : 𝕜) (x
 
 end semi_normed_comm_ring
 
+section normed_field
+variables [normed_field 𝕜] [add_comm_group E] [module 𝕜 E]
+
+private lemma bdd_below_range_add (x : E) (p q : seminorm 𝕜 E) :
+  bdd_below (range (λ (u : E), p u + q (x - u))) :=
+by { use 0, rintro _ ⟨x, rfl⟩, exact add_nonneg (p.nonneg _) (q.nonneg _) }
+
+noncomputable instance : has_inf (seminorm 𝕜 E) :=
+{ inf := λ p q,
+  { to_fun := λ x, ⨅ u : E, p u + q (x-u),
```

Comment:
This is missing an `inf_apply` lemma that states `(p ⊓ q) x = ⨅ u : E, p u + q (x-u)`

Reply:
I know. You said it above and I marked it I resolved because I did the change locally (but not pushed yet), sorry for the confusion